### ISSUE TITLE
My tickets modal view doesn't have Cancel/Dismiss button on iPhone.

### ISSUE THREAD

**Issue by lilyghukasyan** [zendesk/zendesk_sdk_ios#128]
Hi,

My tickets modal view doesn't have Cancel/Dismiss on iPhone app (on iPad "Cancel" button is displayed):
![simulator screen shot dec 3 2015 5 48 55 pm](https://cloud.githubusercontent.com/assets/5071229/11621820/052cf88a-9cd1-11e5-8d67-63753a02f4ba.png)

I use presentRequestListWithNavController: method. 
It's easy to reproduce the issue in the Sample app > ZDSampleViewController > requestListView, if you change showRequestListWithNavController: to presentRequestListWithNavController: and run on iPhone.

thanks.

**Comment by tecknut** [comment]
Hey @lilyghukasyan, we have an internal issue open for this already. Do you need the request list controller to be presented modally? If not you can use the show method instead. 

If you do need it I can give you a quick workaround. 

Thanks.

**Comment by lilyghukasyan** [comment]
Hi @tecknut, 
I do need to present the request list modally, but it is not urgent for now, so I will be looking forward for the fix. Thank you.

**Comment by tecknut** [comment]
@lilyghukasyan If you want to try the workaround, the snippet bellow is all you really need.

```
    ZDKRequestListViewController *requestListController = [ZDKRequestListViewController new];
    requestListController.delegate = [ZDKRequests instance];
```

From this point you can set a left bar button and present the controller manually. 

We should have this patched in an upcoming release.

**Comment by oarrabi** [comment]
Hello @lilyghukasyan. We just released 1.5.4.1 with the fix, please update and check it out.

**Comment by lilyghukasyan** [comment]
Hi @oarrabi , thanks for letting me know.
